dali/operators/decoder/audio/audio_decoder.h

```diff
@@ -21,8 +21,10 @@
 namespace dali {
 
 struct AudioMetadata {
-  int length;
-  int sample_rate;  /// [Hz]
+  /// @brief Length, in samples, of the recording
+  int64_t length;
```

Comment:
It you're talking about the comment, I'll replace it with "(multi-channel) samples", because that's what it is. In libsnd they are using `frames` for multi-channel samples, but that's not that common and even their own documenation uses "multi-channel samples" in some places.